### DISCUSSION TITLE
Combination of options to obtain the yellow button

### DISCUSSION THREAD

**Discussion by Volubyl** [amnestywebsite/humanity-theme#446] [Q&A] [answered]
Hello,

For a project in collaboration with Amnesty International France, I'm currently working with the “Humanity” theme.

It's probably not a real issue because I'm a bit new to Wordpress and I might miss something.

I've installed Wordpress locally with the “Humanity” theme, but I can't seem to create the yellow button that's widely used in Amnesty's graphic charter.

Would you kindly tell me what combination to use in the Wordpress administrator space to get this button?

Thanks in advance, and thanks for your work.

**Comment by jaymcp** [comment, marked as answer]
Hi there

Thank you for your question!

By itself the theme does not include any Amnesty-specific branding. In order to apply the Amnesty branding to the theme, you will need the branding plugin we created. You have been given access to this plugin, so you should receive an email from GitHub with the invitation.

Kind regards
Jay

**Reply by Volubyl** [reply]
Oh, now I understand!

Thanks for your prompt reply!

Do you think it would be worth mentioning in the Read Me ?

Unless I'm the first person to ask that question?

**Reply by jaymcp** [reply]
I think mentioning it in the readme is a good idea; I'll add it to my list :)
